sonar/modules/documents/api.py

```diff
@@ -138,6 +143,118 @@ def get_record_by_identifier(cls, identifiers):
 
         return None
 
+    def add_file(self, data, key, **kwargs):
+        """Create file and add it to record.
+
+        kwargs may contain some additional data such as: file label, file type,
+        order and url.
+
+        :param data: Binary data of file
+        :param str key: File key
+        :returns: File object created.
+        """
+        if not kwargs.get('type'):
+            kwargs['type'] = 'file'
+
+        if not kwargs.get('order'):
+            kwargs['order'] = 1
```

Comment:
What's happens if a file for the same record has already the same order ? Maybe a function `get_next_file_order()` on `DocumentRecord ` class should be better ?

Reply:
Nice recommendation, done.